### DISCUSSION TITLE
MySQL Support

### DISCUSSION THREAD

**Discussion by maebeam** [uptrace/bun#227] [Q&A] [answered]
Hi there,

I'm trying to use `bun` with MySQL and noticing a general lack of support. Is there a general roadmap or resources available for supporting MySQL fully?

I have a branch with some fixes I'm planning on turning into pull requests and I just want to check with the core dev team where MySQL support stands.

Thanks,

Mae

**Comment by vmihailenco** [comment, marked as answer]
Well, most tests are passing for all three supported databases: Postgres, MySQL, and SQLite. MySQL 5.7 does not have CTE and few other features, but it is still usable... We don't aim to support every single MySQL feature, but that is true for Postgres/SQLIte too.

Could you give an example of the feature you are looking for?

>I have a branch with some fixes I'm planning on turning into pull requests and I just want to check with the core dev team where MySQL support stands.

Please send them in :+1:
